web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -290,7 +286,7 @@ describe("profile - shared", () => {
       expect(screen.getByTestId("opp-alert")).toBeInTheDocument();
     });
 
-    it.each(phasesWhereAlertFalse)("does not display alert for %s", (operatingPhase) => {
+    it.each(phasesWhereGoToProfileDoesNotShow)("does not display alert for %s", (operatingPhase) => {
```

Comment:
[pebble] I notice that we no longer have changes to the `profile-owning.test` file. I know there's a ton of tests for the profile alert component in there. Did they get moved to the `ProfileOpportunitiesAlert.test` file?

If so, the ones that are already covered by the new test file should be deleted. And the ones that are NOT covered by the new test file should remain... and maybe they should be relevant across all three personas? For example, "removes question from alert when it gets answered" is important behavior and that has to be tested at the profile level, not the alert level. So it has to stay... and maybe be moved to shared instead of owning because now that's shared behavior across all personas. Whereas "does not display alert if all funding/certification questions are answered" is clearly already covered in ProfileOpportunities alert (well, once the name updates) and should be deleted from the profile-owning test.